### DISCUSSION TITLE
Running a single active job in a distributed environment

### DISCUSSION THREAD

**Discussion by karan-bohra** [taskforcesh/bullmq#2561] [Q&A]
I am using bullmq with nestjs on a distributed instances, I want my queue to run only on only one processor at a time instead of all the instances creating there own jobs.
I couldn't find any options to achieve this in bullmq docs.

**Comment by manast** [comment]
You have multiple workers but you only want to run 1 job at a time ever? I think this is probably the feature you want: https://github.com/taskforcesh/bullmq/pull/2465

**Reply by roggervalf** [reply]
also that pr depends on this one https://github.com/taskforcesh/bullmq/pull/2496
